### DISCUSSION TITLE
How to add functional Google accounts on Skiovox

### DISCUSSION THREAD

**Discussion by synapt0** [3kh0/ext-remover#816] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am unsure on how to add functional Google accounts to the unrestricted browser since whenever I login with one it doesn't show up as an account on the browser. 

I use the 2B method.

**Comment by 3kh0** [comment]
Go to `google.com`, does it show the account being logged in?

**Comment by Su4o** [comment, marked as answer]
on Step 2B.3, dont close out settings. instead login to your account on there. Should work, try at home before trying at school.
![Screenshot 2023-11-21 08 27 09](https://github.com/3kh0/ext-remover/assets/127699325/ba77e9f8-319f-4aa6-9515-d612622d48b6)
![Screenshot 2023-11-21 08 27 44](https://github.com/3kh0/ext-remover/assets/127699325/ac3473d4-66f2-4e16-9c89-8c8563f848e4)
